### PR TITLE
Responsive desktop app

### PR DESCRIPTION
**Pull request by den-cloud**
In this pull I added Media query breakpoints and CSS for desktop responsiveness.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -304,6 +334,159 @@ h2 {
...
+  color: white;
+  background-color: #4472e7;
+  border: 1px solid #4472e7;
+}
+
 .contact-message {
   height: 100px;
 }
+
+@media (min-width: 420px) {
```
Comment (karvel-code) [inline styles.css]: - [x] Kindly ensure that the break point/ `min-width` property for your Desktop ranges between `992px-1100px`.

File: styles.css [hunk 2]
```diff
@@ -304,6 +334,159 @@ h2 {
...
+    display: none;
+  }
+
+  .contact-section {
+    background-image: url("images/contact-desktop-background.png");
+    background-size: cover;
+  }
+}
+
+@media (min-width: 750px) {
```
Comment (karvel-code) [inline styles.css]: - [x] Refer to the above comment on the desktop breakpoint

File: index.html [hunk 3]
```diff
@@ -13,196 +13,217 @@
...
+                    <div>
+                        <div class="work-detail-box">
+                            <ul>
+                                <li class="chips">html</li>
+                                <li class="chips">css</li>
+                                <li class="chips">javascript</li>
+                            </ul>
+                        </div>
+                    </div>
+                    <div class="work-button"><a class="project-button" href="#">See Project</a></div>
```
Comment (karvel-code) [inline index.html:71]: - [x] Please ensure that the `See Project `button redirects the user to this project's live demo or your Github repository. Repeat the same for the rest of the `See Project` buttons.

File: styles.css [hunk 4]
```diff
@@ -304,6 +334,159 @@ h2 {
...
+  .languages-box2 {
+    padding-top: 40px;
+    width: 50%;
+  }
```
Comment (karvel-code) [inline styles.css]: - [x] I'd suggest that you ensure that the container of this section has a width of about 60%. The other section/block should cover around 40% of the width of the page.

File: index.html [hunk 5]
```diff
@@ -13,196 +13,217 @@
...
+                </ul>
+                <div class="work-button"><a class="project-button" href="#">Get My Resume</a></div>
             </div>
-            <div class="language-box">
-                <h3 class="canopy">Skills</h3>
-                <a href="#"><img class="icons" src="images/right-arrow.png" alt="arrow"></a>
+            <div class="languages-box2">
+                <div class="language-box1">
+                    <h3 class="canopy">Languages</h3>
+                    <a href="#"><img class="icons" src="images/down-arrow.png" alt="arrow"></a>
```
Comment (karvel-code) [inline index.html:185]: - [x] Kindly ensure that the arrows for the Languages, Frameworks and Skills are dropdown menus that display in detail the examples of Languages, Frameworks and Skills based on the section.
- [x] Within each drop-down menu, ensure that there are up to 5 tags per row listing the technologies.

**Review by karvel-code — CHANGES_REQUESTED** [review]
Hi @den-cloud , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- All your checks pass successfully.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: styles.css (hunk 1 above)
Comment (den-cloud) [reply styles.css]: If I put that min-width it wont look good , I tried

File: index.html (hunk 5 above)
Comment (den-cloud) [reply index.html:185]: I kindly request that you have a look the the template 1 that I was doing, they're a no rows of technologies that have 5 tags.

File: index.html [hunk 6]
```diff
@@ -13,196 +13,217 @@
...
         <a class="logo" href="#">My Logo</a>
-        <nav class="menu"><img src="images/Union.png" alt="menu"></nav>
+        <nav class="menu"><a href="#"><img src="images/Union.png" alt="menu"></a></nav>
+        <nav class="menu1">
+            <ul>
+                <li><a href="#portfolio">Portfolio</a> </li>
+                <li><a href="#about">About</a></li>
+                <li><a href="#contact">Contact</a></li>
+            </ul>
+        </nav>
```
Comment (Emmyn5600) [inline index.html:23]: - [x] Kindly, change some `padding-top` and `font-size` for our menus to have the same we have on the figma.

----------------------------------------------Yours----------------------------------------------------------
port31
----------------------------------------------Original----------------------------------------------------------
port33

File: styles.css [hunk 7]
```diff
@@ -1,17 +1,31 @@
 @import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap");
 
-body {
+* {
   margin: 0;
   padding: 0;
 }
```
Comment (Emmyn5600) [inline styles.css]: - [x] Kindly, add the `overflow-x:hidden` to remove the scroll left of the page.

port32

File: index.html [hunk 8]
```diff
@@ -13,196 +13,217 @@
...
+                    </li>
+                </ul>
+                <div class="language-box">
+                    <h3 class="canopy">Frameworks</h3>
+                    <a href="#"><img class="icons" src="images/right-arrow.png" alt="arrow"></a>
+                </div>
+                <div class="language-box">
+                    <h3 class="canopy">Skills</h3>
+                    <a href="#"><img class="icons" src="images/right-arrow.png" alt="arrow"></a>
+                </div>
```
Comment (Emmyn5600) [inline index.html:208]: - [x] As suggested by the previous reviewer, kindly work and style the arrow as have on the figma

-----------------------------------------------------Original------------------------------------------------------------

port34


-----------------------------------------------------Yours----------------------------------------------------------------

port35

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @den-cloud 

Great work on implementing the changes requested by the previous reviewer, however there are some issues to be solved before you get approved but you are almost there:muscle:. 
   
Changes required :writing_hand:  

       * #### Please check the comment under the review
       
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html (hunk 6 above)
Comment (den-cloud) [reply index.html:23]: You reviewing me unfair. why can't you understand that the instructions of this project was to try make is the same as possible

Comment (Emmyn5600) [reply index.html:23]: I know you can make it, you only have to have to change the font-size and padding-top value :pray:

File: styles.css (hunk 7 above)
Comment (den-cloud) [reply styles.css]: what do you mean ?

Comment (den-cloud) [reply styles.css]: I am totally confused right now

Comment (Emmyn5600) [reply styles.css]: If you are available, I can help you @den-cloud

Comment (den-cloud) [reply styles.css]: Please do

Comment (Emmyn5600) [reply styles.css]: Just add `overflow-x: hidden` in the body, it will prevent the scroll left for the page.

File: index.html (hunk 6 above)
Comment (den-cloud) [reply index.html:23]: I already did that.
Thanks alot for the review

File: styles.css (hunk 7 above)
Comment (den-cloud) [reply styles.css]: thanks

**Review by Emmyn5600 — APPROVED** [review]
Hi, 
Good job so far! 👏🏼 @den-cloud 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed, however use a descriptive feature branch:heavy_check_mark:
- There are no linter errors:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous TSEs
 
 ## Optional suggestions
 
 - N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**